crates/cheatnet/src/constants.rs

```diff
@@ -46,7 +46,7 @@ pub const TEST_CONTRACT_CLASS_HASH: &str = "0x117";
 pub const STEP_RESOURCE_COST: f64 = 0.01_f64;
 // snforge_std/src/cheatcodes.cairo::test_address
 pub const TEST_ADDRESS: &str = "0x01724987234973219347210837402";
-
+pub const DEFAULT_MAX_TX_STEPS: u32 = 3_000_000;
```

Comment:
What's the justification of this value?

Reply:
https://github.com/orgs/foundry-rs/projects/3/views/4?pane=issue&itemId=40320964

Reply:
Could you link the docs here?